login/FrontQuery.js

```diff
@@ -0,0 +1,7 @@
+class FrontQuery{
```

Comment:
FrontQuery sounds very generic, maybe UserResponse would be more clear and compliant with the conventions?

Reply:
ok,already rename it